src/main/groovy/org/pih/warehouse/outbound/ImportPackingListErrors.groovy

```diff
@@ -0,0 +1,10 @@
+package org.pih.warehouse.outbound
+
+class ImportPackingListErrors {
+
+    Map<String, List<String>> fulfillmentDetails
+
+    Map<String, List<String>> sendingOptions
+
+    Map<String, Map<String, Map<String, List<String>>>> packingList = new HashMap<>()
```

Comment:
This nested map is confusing, perhaps a comment with an example could be useful.

Reply:
I believe I introduced a comment in the controller itself: 
```groovy
// Build errors for each row separately. Row id is the key.
 // e.g. { 2: { binLocation: ["Bin location cannot be null"] } }
errors.packingList[item.rowId] = CommandUtils.buildErrorsGroupedByField(item.errors)
```
but I don't mind adding it also here for clarity